### PR TITLE
Fixed broken links in documentation

### PR DESCRIPTION
**Pull request by wts42**

### CODE REVIEW

File: docs/installation/testing.md [hunk 1]
```diff
@@ -44,6 +44,6 @@ If you don't have any telnet software, these are compatible with ENiGMA½:
 * [SyncTERM](http://syncterm.bbsdev.net/)
 * [EtherTerm](https://github.com/M-griffin/EtherTerm)
 * [NetRunner](http://mysticbbs.com/downloads.html)
-* [MagiTerm](https://magickabbs.com/index.php/magiterm/)
+* [MagiTerm](https://web.archive.org/web/20190330170733/https://magickabbs.com/index.php/2018/01/)
```
Comment (NuSkooler) [inline docs/installation/testing.md:47]: This is the updated link: https://magickabbs.com/utils/

**Comment by NuSkooler** [comment]
I mostly skimmed through, but this all LGTM. I did make one comment RE MagiTerm. Thanks for doing this!

**Comment by wts42** [comment]
> I mostly skimmed through, but this all LGTM. I did make one comment RE MagiTerm. Thanks for doing this!

Glad to be part of this piece of software, and if it's only in documentation. :)
Updated the link to MagiTerm, should be good for merging now.

**Comment by NuSkooler** [comment]
Thank you again, this is great!

**Pull request merged**